ignite/metrics/classification_report.py

```diff
@@ -0,0 +1,107 @@
+import json
+from typing import Callable, Collection, Dict, List, Optional, Union
+
+import torch
+
+from ignite.metrics.fbeta import Fbeta
+from ignite.metrics.metric import Metric
+from ignite.metrics.metrics_lambda import MetricsLambda
+from ignite.metrics.precision import Precision
+from ignite.metrics.recall import Recall
+
+__all__ = ["ClassificationReport"]
+
+
+def ClassificationReport(
```

Comment:
It’s not a valid function name regarding pep8 naming rules.

Reply:
@sdesrozis true, but we are already doing that for IoU, mIoU, Fbeta functions ...

Reply:
> @sdesrozis true, but we are already doing that for IoU, mIoU, Fbeta functions ...

I agree, so I kept it for consistency, otherwise we can rename all of them, but I believe it will be too much work.